test/metric_dimension/agent_configs/drop_original_metrics.json

```diff
@@ -0,0 +1,47 @@
+{
+  "agent": {
+    "metrics_collection_interval": 10
+  },
+  "metrics": {
+    "force_flush_interval": 5,
+    "namespace": "TestDropOriginalMetrics",
+    "append_dimensions": {
+      "InstanceId": "${aws:InstanceId}",
+      "InstanceType": "${aws:InstanceType}"
+    },
+    "aggregation_dimensions": [
+      [],
+      [
+        "InstanceId"
+      ],
+      [
+        "InstanceId",
+        "InstanceType"
+      ]
+    ],
+    "metrics_collected": {
+      "mem": {
+        "drop_original_metrics": ["*"],
+        "measurement": [
+          "available",
+          "used_percent"
+        ]
+      },
+      "cpu": {
+        "drop_original_metrics": ["usage_guest", "usage_idle"],
+        "measurement": [
+          "usage_guest",
```

Comment:
Can we do a rename for one these?

Reply:
Can add that after. Didn't want to make the PR too busy.

Reply:
Added the rename.